team13phase1.tex

```diff
@@ -56,6 +56,24 @@
     \end{fr}
   \end{fr}
 \item
+  The system shall interact with external, Internet-facing services and devices.
+  \begin{fr}
+  \item\label{fr:external:weather}
+    The system shall incorporate local sensors and data from Internet-based weather services to make weather- and temperature-dependent decisions.
+  \end{fr}
+\item
+  The system shall monitor devices internal to it.
+  \begin{fr}
+  \item
+    The system shall monitor the amount of food in the refrigerator.
+    \begin{fr}
+    \item
+      The system shall be able to determine when the refrigerator is empty.
+    \end{fr}
+  \item\label{fr:internal:smoke_detector}
+    The system shall monitor properties' smoke detectors.
```

Comment:
FRs like these where something is being monitored, watched, or polled need to have an indicator and an reaction to that indicators. For example, `The system shall monitor each property's smoke detectors and send a high priority alert to the property manager when smoke is detected.` You might want to combine some of the FRs from the WLAN section to here, so that these can have more meaning.

Reply:
But the monitoring and the notifying are two discrete functionalities, are they not?  Certainly the latter depends on the former, but I think that should be expressed by referencing the former not by combining both pieces of functionality into a single FR.